server/tests/src/test/java/org/infinispan/server/functional/AbstractMultiClusterIT.java

```diff
@@ -29,11 +29,13 @@
 import org.infinispan.server.test.core.TestSystemPropertyNames;
 import org.infinispan.util.KeyValuePair;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Tag;
 
 /**
  * @author Ryan Emerson
  * @since 12.0
  */
+@Tag("embedded")
```

Comment:
call it "embeddedserver" please, otherwise it's a bit ambiguous with real embedded mode :)